### PR TITLE
Trranslate CDL to CXF

### PR DESCRIPTION
**Pull request by anandkp92**
* Adding a new output format `-o cxf`, which produces a `.jsonld` representation of a CDL sequence
* Based on ASHRAE Standard 231P
* This version based on public review draft of the ASHRAE S231P
* The CXF translation will continue to be updated as we incorporate more comments from ASHRAE S231P

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,22 +112,43 @@ Specify the output directory. The default option is the current directory.
...
 
 Graphical viewers are available (please use right click + open in a new tab or refresh the page if necessary - this is not optimized for Firefox) :
-- [CDL Schema viewer](cdl-viz.html)
-- [Modelica Schema viewer](modelica-viz.html)
+- [CXF Schema viewer](https://htmlpreview.github.io/?https://github.com/lbl-srg/modelica-json/blob/issue214_cxf/cxf-viz.html)
+- [Modelica Schema viewer](https://htmlpreview.github.io/?https://github.com/lbl-srg/modelica-json/blob/master/modelica-viz.html)
+
+## 5. CXF-Core.jsonld
+
+[CXF-Core.jsonld](CXF-Core.jsonld) contains the CXF representation of all CDL elementary blocks, classes and relationships: It can be used to validate a CXF sequence.
```
Comment (JayHuLBL) [inline README.md:143]: how can the CXF representation of the CDL elementary blocks be used to validate the CXF sequences?

Comment (anandkp92) [reply README.md:143]: We can use the CXF-Core.jsonld to make sure that the CXF sequence aligns with the standard (make sure they inputs and outputs of elementary blocks used make sense etc.)

Comment (anandkp92) [reply README.md:143]: But maybe I'll remove it now - we can add it once we have a validation tool.

File: app.js [hunk 2]
```diff
@@ -58,7 +59,25 @@ parser.addArgument(
   ['-p', '--prettyPrint'],
   {
     help: 'Pretty print JSON output.',
```
Comment (JayHuLBL) [inline app.js:61]: `'Pretty print JSON output. The -o/--output should be 'json' or 'raw-json`.`

**Review by JayHuLBL — APPROVED** [review]

**Pull request merged**